app/src/main/res/layout/activity_notification.xml

```diff
@@ -47,7 +47,8 @@
                 app:layout_constraintRight_toRightOf="parent"
                 app:layout_constraintTop_toTopOf="parent"
                 app:layout_constraintBottom_toBottomOf="parent"
-                app:srcCompat="@drawable/ic_notifications_off_black_24dp" />
+                app:srcCompat="@drawable/ic_notifications_off_black_24dp"
+                android:contentDescription="@string/no_notification" />
```

Comment:
The string used here says "You have no unread notifications" and this does not describe the UI element. It is the notification text already.